playlist/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+from users.models import User
+
+
+class Playlist(models.Model):
+    """Songlist
+    """
+    title = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+    date_created = models.DateTimeField(auto_now_add=True)
+    date_updated = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return ("{}".format(self.title))
+
+
+class Song(models.Model):
+    """Song
+    """
+    playlist = models.ForeignKey(Playlist)
+    title = models.CharField(max_length=255)
+    link = models.URLField()
+    user = models.ForeignKey(User)
+    date_created = models.DateTimeField(auto_now_add=True)
+    date_updated = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return ("{}".format(self.title))
```

Comment:
the function name says __str__ so maybe it wants to return a string not whatever you put it

Reply:
In this case it does nothing. Sorry sir going to remove it.